dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -494,5 +494,9 @@ schemagen.info.schema.file.created=CWWKG0109I: The schema was created in the [{0
 schemagen.info.schema.file.created.explanation=The command completed successfully.
 schemagen.info.schema.file.created.useraction=No action is required.
 
+error.config.root.deleted=CWWKG0110E: A configuration update was detected, but the configuration root {0} does not exist. No configuration updates were made.
```

Comment:
Do we want to state the "configuration root" is gone?  or the server.xml file is gone?   not sure if users will know what it means describing the configuration root...